<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DLG_GateInfo</class>
 <widget class="QWidget" name="DLG_GateInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>140</width>
    <height>175</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="btn_DeleteGate">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>5</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../resouces.qrc">
     <normaloff>:/Resources/Button Icons/icon-bin.png</normaloff>:/Resources/Button Icons/icon-bin.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>22</width>
     <height>22</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_Type">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Type :</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_GateType">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="signalCheck">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>50</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Input Signal</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_Frequency">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_Frequency">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Frequency</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Edit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>5</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../resouces.qrc">
     <normaloff>:/Resources/Button Icons/gate-collection-optimize.png</normaloff>:/Resources/Button Icons/gate-collection-optimize.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>22</width>
     <height>22</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="Line" name="line_divider">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>118</width>
     <height>3</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_pageUpdateFrequency">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_pageUpdateFrequency">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Page Update Frequency</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_save">
   <property name="geometry">
    <rect>
     <x>105</x>
     <y>40</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../resouces.qrc">
     <normaloff>:/Resources/Button Icons/icon-file-down.png</normaloff>:/Resources/Button Icons/icon-file-down.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>22</width>
     <height>22</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_drag">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../resouces.qrc">
     <normaloff>:/Resources/Button Icons/icon-drag.png</normaloff>:/Resources/Button Icons/icon-drag.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>22</width>
     <height>22</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_truthTable">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../resouces.qrc">
     <normaloff>:/Resources/Button Icons/gate-collection-truth-table.png</normaloff>:/Resources/Button Icons/gate-collection-truth-table.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>22</width>
     <height>22</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_expression">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>40</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="../resouces.qrc">
     <normaloff>:/Resources/Button Icons/gate-collection-boolean-expressions.png</normaloff>:/Resources/Button Icons/gate-collection-boolean-expressions.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>22</width>
     <height>22</height>
    </size>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../resouces.qrc"/>
 </resources>
 <connections/>
</ui>
